packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -412,6 +413,112 @@ Content-Type: application/json;charset=UTF-8
+
+>  **Note:** This endpoint's base URL varies depending on whether you are using a Custom Authorization Server. For more information, see [Composing your base URL](#composing-your-base-url).
+
+#### Request parameters
+
+The following parameters can be posted as a part of the URL-encoded form values to the API.
+
+> **Note:** The `/bc/authorize` endpoint requires client authentication. See the [Client authentication methods](#client-authentication-methods) section for more information on which method to choose and how to use the parameters in your request.
+
+| Parameter               | Description                                                                                                                                                                                     | DataType | Required |
+| :---------------------- | :----------------------------------------------------------------------------------------------------------------------------                                                                   | :-----   | :-----   |
+| scope                   | `openid` is required for authentication requests. Other [scopes](#access-token-scopes-and-claims) may also be included.                                                                         | String   | TRUE     |
+| login_hint              | A hint to the OpenID Provider regarding the end-user for whom authentication is being requested.                                                                                                | String   | TRUE     |
+| id_token_hint           | An ID Token previously issued to the Client as a hint to identify the end-user for whom authentication is being requested.                                                                      | String   | TRUE     |
+| binding_message         | A message which is displayed for the user to identify the transaction.                                                                                                                          | String   | FALSE    |
+| request_expiry          | A positive integer allowing the client to request the expires_in value for the auth_req_id the server will return. Max value is 5 minutes                                                       | Integer  | FALSE    |
+| request                 | A JWT created by the client that enables requests to be passed as a single, self-contained parameter. See [Parameter details](#client-initiated-backchannel-authentication-parameter-details).  | JWT      | FALSE    |
+
+>  **Note:** Only one of `login_hint` or `id_token_hint` must be specified in the authentication request.
+
+#### Client-Initiated Backchannel Authentication Parameter details
+
+* `request`:
+
+  * You must sign the JWT using either the app's client secret or a private key whose public key is registered on the app's JWKSet.
+  * The JWT can't be encrypted.
```

Comment:
does this mean that PKCE isn't supported?
```suggestion
  * The JWT can't be encrypted.
```

Reply:
No, this is only talking about the signed requests.